src/tasks/patreon.ts

```diff
@@ -31,6 +31,7 @@ patreonApiURL.search = new URLSearchParams([
 ]).toString();
 
 export const tiers: [PatronTierID, BitField][] = [
+	[PatronTierID.Six, BitField.IsPatronTier5],
```

Comment:
is this meant to be t6 = t5?

Reply:
No, because there is no PatreonTier6 as far as the bot is concerned, it maps to Tier5, just like the github t6 does.